src/system/SystemLayer.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file implements LayerImplLwIP using LwIP.
```

Comment:
please stop using `@file` comments in the source code. They very easily suffer from copy & paste inaccuracies (this seems like one of those instances) and provide no useful info to code readers, often  just restating the file name and using up vertical space.